index.html

```diff
@@ -0,0 +1,1048 @@
+                <div class="article__image-container">
+                  <img
+                    src="./images/articles/article-laurent.webp"
+                    alt="article laurent"
+                    width="624"
+                    height="289"
+                    class="article__big-image"
+                  />
+                  <span class="article__label label"> tag </span>
+                </div>
+                <div class="article__article-information list list__column">
+                  <h3 class="article-information__title heading__article-title">
+                    Best Dishes
+                  </h3>
+                  <time
+                    class="article-information__description"
+                    datetime="2024-05-05"
+                  >
+                    5 may 2024
+                  </time>
+                </div>
+              </article>
+            </a>
+          </li>
+          <li>
+            <a href="./blog-details.html" class="articles__link">
+              <article class="articles-list__article">
+                <div class="article__image-container">
+                  <img
+                    src="./images/articles/article-lotus.webp"
+                    alt="article lotus"
+                    width="624"
+                    height="289"
+                    class="article__big-image"
+                  />
+                  <span class="article__label label"> tag </span>
+                </div>
+                <div class="article__article-information list list__column">
+                  <h3 class="article-information__title heading__article-title">
+                    Contact Us
+                  </h3>
+                  <time
+                    class="article-information__description"
+                    datetime="2024-06-10"
+                  >
+                    10 june 2024
+                  </time>
+                </div>
+              </article>
+            </a>
+          </li>
+        </ul>
+      </section>
+    </main>
+    <footer class="footer list list__column">
+      <div class="footer__summary list">
+        <div class="summary__information list list__column">
+          <a href="./index.html" class="information__link">
+            <strong class="information__title">
+              <img
+                src="./images/icons/logo.svg"
+                alt="Pixmart logo"
+                width="300"
+                height="58"
+                class="information__logo"
+              />
+            </strong>
+          </a>
+          <p class="information__description">
+            Your premium digital assets solution with high quality and
+            comprehensive range of products and services.
+          </p>
+        </div>
+        <div class="summary__summary-lists">
+          <h2 class="summary-lists__title heading__card-title">Company</h2>
+          <nav class="summary-lists__navigation">
```

Comment:
[Usage notes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes)
It's not necessary for all links to be contained in a <nav> element. <nav> is intended only for a major block of navigation links; typically the [<footer>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) element often has a list of links that don't need to be in a <nav> element.

Reply:
Fixed 🥇